recipes/pyopenrivercam/meta.yaml

```diff
@@ -0,0 +1,86 @@
+{% set name = "pyopenrivercam" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/localdevices/pyorc/archive/refs/tags/v0.2.1.tar.gz
+  sha256: 985d4afff0a38384ead1076c89e1a86c75225b4f19bb00d1ee57e324caff8ab2
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ cdt('mesa-libgl-devel') }}  # [linux]
+    - {{ cdt('mesa-libegl-devel') }}  # [linux]
+    - {{ cdt('mesa-dri-drivers') }}  # [linux]
+    - {{ cdt('libselinux') }}        # [linux]
+    - {{ cdt('libxdamage') }}        # [linux]
+    - {{ cdt('libxfixes') }}         # [linux]
+    - {{ cdt('libxxf86vm') }}        # [linux]
+
+  host:
+    - python >=3.8
+    - pip
+    - ffmpeg                         # [not win]
+    - glib                           # [unix]
+    - xorg-libxfixes                 # [linux]
+
+  run:
+    - python >=3.8
+    - cartopy
+    - descartes
+    - geojson
+    - geopandas
+    - geos
+    - gitpython
+    - jupyter
```

Comment:
I will resolve requests, watermark and pytest, and add dask. cartopy and geopandas are very difficult to install through `pip`, therefore I omitted them there, but I do want them included in this recipe. Is this ok?

Reply:
Usually we only specify the dependencies required by the package. Other useful packages can be installed in the same environment with conda. But if this package is always used with `cartopy` and `geopandas` I think it's fine to add them in the recipe.

Reply:
`cartopy` and `geopandas` are used in the plotting functionalities, so they should really be there.
I checked why I added the `requests`, `watermark` and `pytest` requirements. It is because apparently the `openpiv` package requires them. If you do not add these, installation will seem successful, but when importing `openpiv`, the user will get an error, that these dependencies are missing. I suggest that we therefore leave these in.